python/ray/data/tests/test_dataset_formats.py

```diff
@@ -2864,6 +2867,60 @@ def get_node_id():
     assert node_ids == {bar_node_id}
 
 
+def test_image_folder_datasource(ray_start_regular_shared):
```

Comment:
Move these tests above line 2735 as they use the `ray_start_regular_shared` fixture.

Reply:
fixed